File: S13_20-10-2022/constructor.js
class Student {
  name;
  city;

  constructor(name, city) {
    this.name = name;
    this.city = city;
  }

  display() {
    console.log(this.name, this.city);
  }
}

var s1 = new Student("Rahul", "Bhopal");
var s2 = new Student("Ziya", "Mumbai");
// s1.name = "Raj";
// s1.city = "Delhi";

// s2.name = "Sneha";
// s2.city = "Banglore";

s1.display();
s2.display();

File: S21_09-11-2022/Constructor.java
public class Constructor {

    int num1;
    int num2;

    public Constructor() {
        num1 = 100;
        num2 = 200;
    }

    public Constructor(int x, int y) {
        num1 = x;
        num2 = y;
    }

    public void display() {
        System.out.println(num1);
        System.out.println(num2);
    }

    public void print() {
        display();
    }

    public static void main(String Args[]) {
        Constructor c1 = new Constructor(30, 40);
        Constructor c2 = new Constructor();

        // c1.display();
        c1.print();
        c2.print();
    }
}
